src/asInput/index.jsx

```diff
@@ -149,21 +163,28 @@ const asInput = (WrappedComponent, labelFirst = true) => {
     render() {
       const { description, error, describedBy } = this.getDescriptions();
       return (
-        <div className={styles['form-group']}>
-          {labelFirst && <label id={`label-${this.state.id}`} htmlFor={this.state.id}>{this.props.label}</label>}
+        <div className={[classNames({
+            [styles['form-group']]: inputType !== 'checkbox',
```

Comment:
All these checks for `inputType === 'checkbox'` feel a little brittle to me. Could you maybe encapsulate this check as a method like, say, `isGroupedInput`? That way we could easily update it when/if we pass `RadioButton` through `asInput` as well.

Reply:
Good call -- I'll move that through and get the PR updated shortly